remove preQualification extension, update framework reference (#245)
open-contracting-extensions/public-private-partnerships: docs/framework.md

## II. Procurement Information

> Dates and summary details, links to all procurement documents, final feasibility study, including land acquisition, social, environmental, and rehabilitation related information, reports of independent procurement auditors (if any)

Procurement procedures can involve one or more competitive stages:

* In a single-stage procedure, the procuring entity invites suppliers to bid without submitting any prior information.

> * In a multi-stage procedure, suppliers submit requests to participate in a first stage. The procuring entity assesses these requests, and establishes a list of qualified suppliers to invite to bid. Qualified supplies then submit bids in a second stage. The procuring entity assesses the bids, and awards a contract to the winning bid.

Reviewer: ```suggestion
* In a single-stage procedure, the procuring entity invites suppliers to submit bids, without submitting any prior information.
* In a multi-stage procedure, the procuring entity invites suppliers to submit requests to participate in a first stage (pre-qualification). The procuring entity assesses the requests, and establishes a list of qualified suppliers to invite to submit bids. Qualified suppliers then submit bids in a second stage. The procuring entity assesses the bids, and awards a contract to the winning bidder.
```

> In OCDS, the `tender` section represents the invitation to participate and is also used to describe the procedure as a whole.

Reviewer: ```suggestion
In OCDS, the `tender` section represents the invitation to participate (whether to submit bids, or to submit requests to participate) and is also used to describe the procedure as a whole.
```

### II.1. First stage

> The act of testing prospective bidders to determine whether they meet the pass/fail qualification criteria in advance of issuing the request for proposals. ([Source](https://ppp-certification.com/ppp-certification-guide/glossary))

> Use the `tender` section to describe the first stage of the contracting process, whether that is the tender itself, or a pre-qualification stage.

Reviewer: ```suggestion
Use the `tender` section to describe the first stage of the contracting process, whether it involves the submission of bids, or requests to participate.
```

</div>

## II. Procurement Information

> Dates and summary details, links to all procurement documents, final feasibility study, including land acquisition, social, environmental, and rehabilitation related information, reports of independent procurement auditors (if any)

Procurement procedures can involve one or more competitive stages:

* In a single-stage procedure, the procuring entity invites suppliers to bid without submitting any prior information.

> * In a multi-stage procedure, the procuring entity issues an invitation to participate in the process based on which suppliers submit requests to participate. The procuring entity then assesses the submissions and establishes a list of qualified suppliers to invite to bid.

Reviewer: This paragraph is confusing.

```suggestion
* In a multi-stage procedure, suppliers submit requests to participate in a first stage. The procuring entity assesses these requests, and establishes a list of qualified suppliers to invite to bid. Qualified supplies then submit bids in a second stage. The procuring entity assesses the bids, and awards a contract to the winning bid.
```

</div>

Use the `tender/tenderPeriod` field to provide the period during which the first stage is open for submissions. `tenderPeriod/endDate` should contain the closing date for submissions.

> **Schema**: Information can be provided using a [Period](https://standard.open-contracting.org/1.1/en/schema/reference/#period) object in the [tender/period](../reference/schema/#release-schema.json,,tender/period) field.

Reviewer: ```suggestion
**Schema**: Information can be provided using a [Period](https://standard.open-contracting.org/1.1/en/schema/reference/#period) object in the [tender/tenderPeriod](../reference/schema/#release-schema.json,,tender/tenderPeriod) field.
```

</div>

> Use the `tender/period` field to provide the period during which the first stage is open for submissions, `period/endDate` should contain the closing date for submissions.

Reviewer: ```suggestion
Use the `tender/period` field to provide the period during which the first stage is open for submissions. `period/endDate` should contain the closing date for submissions.
```

```eval_rst
.. jsoninclude-flat:: examples/full.json
  :jsonpointer: /releases/1/tender
  :include_only: eligibilityCriteria
  :ignore_path: /releases/1/
```

</div>

#### II.1.6. Request for Qualification

> Request for Qualification (RFQ):

Reviewer: Do we need this line, which repeats the heading?

Author: It's in keeping with the pattern used in other headings: provide the name used in the framework under the OCDS for PPPs heading. Although, in this case, the two are identical. Happy to remove it if you prefer?

Reviewer: I suggest either removing it or putting it inside the following blockquote, so that it's clear it is being quoted from the framework.

Author: Removed in https://github.com/open-contracting-extensions/public-private-partnerships/pull/245/commits/ef0651de03805ab5f86c3493e16fb4452b0422ae

```eval_rst
.. jsoninclude-flat:: examples/full.json
  :jsonpointer: /releases/7/tender
  :include_only: eligibilityCriteria
  :ignore_path: /releases/7/
```

</div>

#### II.1.6. Request for Qualification documents

Request for Qualification (RFQ):

> The set of documents issued by the procuring authority that constitute the basis of the qualification and potentially the pre-selection of candidates (the short list). Qualified (or short-listed candidates) will then be invited to submit a proposal (or to enter into a new phase prior to bid submission, such as a dialogue phase or interactive phase). ([Source](https://ppp-certification.com/ppp-certification-guide/glossary))

> In a single-stage procedure, the RFQ documents are embedded in the Request for Proposal documents.

Reviewer: What is the source of this statement? I wasn't aware that an RFQ could be part of an RFP; typically these are separate stages/processes.

Author: See the entry for Request For Proposal in https://ppp-certification.com/ppp-certification-guide/glossary

Reviewer: Aha. NIGP describes RFP and RFQ as separate sub-processes: https://www.nigp.org/resource/global-best-practices/Public-Private%20Partnership%20P3%20Facilities%20and%20Infrastructure%20Best%20Practice.pdf?dl=true

What APMG above is saying is that qualification/selection criteria are included in the RFP documents. I think it's clearer to say that, than to say "RFQ documents are embedded in the RFP documents".

Reviewer: Or, this statement from APMG for "One-stage tender" is clearer:

> A tender process in which the government issues its Request for Qualifications (RFQ) and Request for Proposal (RFP) together, and bidders submit their qualifications and bid as a single submission.

The original statement from AMPG for "Request for Proposal (RFP)" was:

> In one-stage open tender processes, the contents of a Request for Qualifications (RFQ) are embedded in the same set of documents, for example, the Request for Proposal (RFP) also includes the RFQ regulations.

This is somewhat different than the profile's text, because "documents embedded in documents" doesn't make sense, but "the contents of a Request for Qualifications (RFQ) are embedded", or "the Request for Proposal (RFP) also includes the RFQ regulations" does make sense.

Author: Good point. I updated the RFQ guidance in https://github.com/open-contracting-extensions/public-private-partnerships/pull/245/commits/475291c062d8b7ed4c3179258825bd557fba155e and added guidance for the RFP, which was missing from the revised guidance, in https://github.com/open-contracting-extensions/public-private-partnerships/pull/245/commits/7a95a3c70b5e7b40b5193b2e90cb962cd09842f6.

</div>

> Use [Document](reference/documents) objects in the [tender/documents](../reference/schema/#release-schema.json,,tender/documents) array to provide links to procurement documents, feasibility studies, including land acquisition, social, environmental, and rehabilitation related information and reports of independent procurement auditors. A short summary of each document can be provided using its `description` field. Each document's `documentType` field should be set to a value from the [document type codelist](../reference/codelists/#documenttype), to identify the type of document being disclosed.

Reviewer: The "including land acquisition, social, environmental, and rehabilitation related information" in the middle of a list of document types reads a bit odd.

Author: Yes, it's based on this part of the framework which also reads oddly:

> links to all procurement documents, final feasibility study(including cost-benefit analysis, if any), land acquisition, rehabilitation, social (including information on human rights impact assessment) and environmental assessment, related information, and reports of independent procurement auditors (if any).

How about:

```markdown
Use [Document](reference/documents) objects in the [tender/documents](../reference/schema/#release-schema.json,,tender/documents) array to provide links to documents.

You should provide links to:

* all procurement documents 
* feasibility studies, including cost-benefit analyses, if available
* land acquisition, rehabilitation, social and environmental assessments, including information on human rights impact assessments
* reports of independent procurement auditors

A short summary of each document can be provided using its `description` field. Each document's `documentType` field should be set to a value from the [document type codelist](../reference/codelists/#documenttype), to identify the type of document being disclosed.
```

Reviewer: That's much clearer. I think we can change one bullet to:

```
* land acquisition, rehabilitation, social, human rights, and environmental assessments
```

The original wording I guess treats human rights assessments as a sub-type of social assessment, but I don't think this concept hierarchy is important to retain.

A clearer, easier to read, but longer format would be:

```
* land acquisition assessments
* rehabilitation assessments
* social assessments
* human rights assessments
* environmental assessments

Author: Updated in https://github.com/open-contracting-extensions/public-private-partnerships/pull/245/commits/ef0651de03805ab5f86c3493e16fb4452b0422ae

**Example**: See [section I.4](#i-4-project-economic-and-social-benefits) for JSON and flattened examples of the `documents` building block.

### II.2. List of pre-qualified suppliers

> Pre-qualification or shortlist.

Reviewer: Should this be:

```suggestion
> Pre-qualification or shortlist.
```

Author: I don't think so. The names from the framework provided under other headings aren't quoted. Only definitions from other sources are quoted. However, perhaps we could make it clearer that the names are taken directly from the framework.

Reviewer: I think it would be clearest for them all to be indicated somehow. I think quoting is fine for that purpose.

Author: In https://github.com/open-contracting-extensions/public-private-partnerships/pull/245/commits/ef0651de03805ab5f86c3493e16fb4452b0422ae I've quoted all definitions from the framework and removed any which simply repeat the headings.

Reviewer: I think maybe this specific one isn't quoted yet.
